lib/trino-filesystem-azure/src/main/java/io/trino/filesystem/azure/AzureLocation.java

```diff
@@ -125,7 +125,9 @@ public String toString()
 
     public Location baseLocation()
     {
-        return Location.of("abfs://%s%s.dfs.core.windows.net/".formatted(
+        String scheme = location.scheme().orElse("abfs");
```

Comment:
Location's scheme is validated in the constructor, so it can never be empty. Should we save it in a property?